pika/adapters/blocking_connection.py

```diff
@@ -1528,7 +1528,7 @@ def basic_consume(self,
                 method: spec.Basic.Deliver
                 properties: spec.BasicProperties
                 body: str or unicode
-        :param bool no_ack: Tell the broker to not expect a response (i.e.,
+        :param bool auto_ack: Tell the broker to not expect a response (i.e.,
           no ack/nack)
```

Comment:
Optional: Wherever we document the `auto_ack` arg, it might be helpful to note that it correlates to AMQP no-ack